ts/features/itwallet/analytics/index.ts

```diff
@@ -131,29 +135,48 @@ type TrackITWalletSpidIDPSelected = { idp: string };
 
 type TrackItWalletCieCardReadingFailure = { reason: string };
 
+export type ItwCredentialMixpanelStatus =
+  | "not_available"
+  | "valid"
+  | "not_valid"
+  | "expiring"
+  | "expired"
+  | "expiring_verification"
+  | "verification_expired"
+  | "unknown";
+
 export type ItwStatus = "not_active" | "L2" | "L3";
-export type ItwId = "not_available" | "valid" | "not_valid";
-export type ItwPg = "not_available" | "valid" | "not_valid" | "expiring";
-export type ItwTs = "not_available" | "valid" | "not_valid" | "expiring";
-export type ItwCed = "not_available" | "valid" | "not_valid" | "expiring";
+export type ItwId = "not_available" | "valid" | "expiring" | "expired";
+export type ItwPg = ItwCredentialMixpanelStatus;
+export type ItwTs = ItwCredentialMixpanelStatus;
+export type ItwCed = ItwCredentialMixpanelStatus;
```

Comment:
Do we need `ItwPg`, `ItwTs`, `ItwCed` if they are the same as `ItwCredentialMixpanelStatus`? Do they add additional information? 
Also, `ItwId` could be extracted from  `ItwCredentialMixpanelStatus`. What do you think?